tests/features/observation_steps.feature

```diff
@@ -23,7 +23,7 @@ Scenario: I see user errors
 @slow @o3
 Scenario: I check navigation
   Given I start the interview at "all_tests"
-  And the user gets to "showifs" with this data:
+  And I get to "showifs" with this data:
```

Comment:
Since the tests now support both sentences ("the user gets..." and "I get..."), should one of the tests still retain "the user gets..." language to make sure that is still tested?

Reply:
Good point. It should probably be its own test. Made #292. In fact, we should figure out all the variations possible in these steps and test all of them. Really good point.